test/test-stream.js

```diff
@@ -169,3 +169,27 @@ assert(new Iconv('ascii', 'ascii') instanceof stream.Stream);
   stream.end('b2s=', 'base64');
   assert(ok);
 })();
+
+(function () {
+    var ok = false;
+    var stream = Iconv('gb18030', 'utf-8');
+    stream.once('error', function (e) {
+        assert.equal(e.message, 'Incomplete character sequence.');
+        assert.equal(e.code, 'EINVAL');
+        ok = true;
+    });
+    stream.end(new Buffer([0x00, 0xf1, 0x52, 0x00, 0x00, 0x78, 0x51, 0xd9, 0xf7, 0x78, 0x51, 0xd9]));
```

Comment:
Long line, please wrap at 80 columns.  Maybe write it as:

    var octets = [0x00,0xf1,0x52,0x00,0x00,0x78,0x51,0xd9,0xf7,0x78,0x51,0xd9];
    stream.end(new Buffer(octets));